doc/CODING.md

```diff
@@ -104,6 +109,7 @@ Use `\copydoc`, `\copybrief` and `\copydetails` intensively.
 
 Files should start with:
 
+\verbatim
```

Comment:
doxygen parses markdown files first with the doxygen parser and afterwards with the markdown parser, so it recognizes the @commands as doxygen commands. This behaviour produces an error and therefore is no output between /** and */ in the doxygen html. I did not find a nice solution for this problem :/